solutions/src/main/java/ru/sberbank/school/task02/services/implementation/ServiceFactoryImpl.java

```diff
@@ -1,26 +1,38 @@
 package ru.sberbank.school.task02.services.implementation;
 
+import lombok.NonNull;
 import ru.sberbank.school.task02.ExtendedFxConversionService;
 import ru.sberbank.school.task02.ExternalQuotesService;
 import ru.sberbank.school.task02.FxConversionService;
 import ru.sberbank.school.task02.ServiceFactory;
 import ru.sberbank.school.task02.services.InternalQuotesService;
 
+/**
+ * Фабрика сервиса - валютного калькулятора.
+ */
 public class ServiceFactoryImpl implements ServiceFactory {
+
     /**
      * Возвращает инстанс FxConversionService реализованнную студентом.
      *
      * @param externalQuotesService Сервис-источник актуальных котировок
      * @return объект - валютный калькулятор
      */
     @Override
-    public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) {
+    public FxConversionService getFxConversionService(@NonNull ExternalQuotesService externalQuotesService) {
         InternalQuotesService internalQuotesService = new InternalQuotesServiceImpl(externalQuotesService);
         return new FxConversionServiceImpl(internalQuotesService);
     }
 
+    /**
+     * Возвращает инстанс ExtendedFxConversionService.
+     *
+     * @param externalQuotesService Сервис-источник актуальных котировок
+     * @return объект - обратный валютный калькулятор
+     */
     @Override
-    public ExtendedFxConversionService getExtendedFxConversionService(ExternalQuotesService externalQuotesService) {
-        return null;
+    public ExtendedFxConversionService getExtendedFxConversionService(@NonNull ExternalQuotesService externalQuotesService) {
```

Comment:
Line is longer than 120 characters (found 125).

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1555567973>*